Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -140,7 +148,11 @@ enum PackageShow {
                     .hr(
                         .class("minor")
                     ),
-                    sidebarInfoForPackageAuthors()
+                    sidebarInfoForPackageAuthors(),
+                    .hr(
+                        .class("minor")
+                    ),
+                    sidebarPublishedAndUpdatedSection()
```

Comment:
I like the experimentation with giving a couple of spots for this to be positioned, but since this information is effectively just for Google I think we should put it under the readme and remove this one.

Of course, once it's no longer in the sidebar it'll need a more appropriate name, too. We could even call it something like `visibleMetadataSection`so it's obvious what the purpose of it is so it stops us from adding more to it in the future (unless we need more visible metadata later). We should also probably add a comment to it to explain its purpose, maybe even with a link to the Google documentation?

Reply:
Should this content be within the `article` tag or not. Also Google indicates we should…

> feature it prominently.

What does that even mean. 🤣

Reply:
Yea Google is always vague to attempt to stop gaming their algorithm. I think it'll be fine at the bottom but we'll know for sure because this change is going to make a visible change to search results, like this:

![Screenshot 2022-04-19 at 12 47 59@2x](https://user-images.githubusercontent.com/5180/163996839-02d61c6e-9c3a-4009-80f0-e65aea232963.png)

So if that doesn't appear after a week or two of this being deployed, we'll need to move it.